addons/web/static/src/views/view_components/view_scale_selector.js

```diff
@@ -16,6 +16,7 @@ export class ViewScaleSelector extends Component {
         isWeekendVisible: { type: Boolean, optional: true },
         setScale: { type: Function },
         toggleWeekendVisibility: { type: Function, optional: true },
+        dropdownClasses: { type: String, optional: true },
```

Comment:
Thanks for the fix. Can you just rename dropdownClasses to `dropdownClass`? To have a similar name with the props of the dropdown togglerClass

Once it’s done I’ll approve it